diff --git a/src/mcl_c.js b/src/mcl_c.js
index 9c33108..025e186 100644
--- a/src/mcl_c.js
+++ b/src/mcl_c.js
@@ -107,11 +107,11 @@ if (ENVIRONMENT_IS_NODE) {
  if (_scriptDir) {
   scriptDirectory = _scriptDir;
  }
- if (scriptDirectory.indexOf("blob:") !== 0) {
-  scriptDirectory = scriptDirectory.substr(0, scriptDirectory.replace(/[?#].*/, "").lastIndexOf("/") + 1);
- } else {
-  scriptDirectory = "";
- }
+// if (scriptDirectory.indexOf("blob:") !== 0) {
+//  scriptDirectory = scriptDirectory.substr(0, scriptDirectory.replace(/[?#].*/, "").lastIndexOf("/") + 1);
+// } else {
+//  scriptDirectory = "";
+// }
  {
   read_ = url => {
    try {
@@ -352,6 +352,7 @@ function getBinary(file) {
 }

 function getBinaryPromise() {
+/*
  if (!wasmBinary && (ENVIRONMENT_IS_WEB || ENVIRONMENT_IS_WORKER)) {
   if (typeof fetch == "function" && !isFileURI(wasmBinaryFile)) {
    return fetch(wasmBinaryFile, {
@@ -374,6 +375,7 @@ function getBinaryPromise() {
    }
   }
  }
+*/
  return Promise.resolve().then(function() {
   return getBinary(wasmBinaryFile);
  });
